aifc/aifc/__init__.py

```diff
@@ -141,7 +141,6 @@
 __all__ = ["Error", "open"]
 
 
-warnings._deprecated(__name__, remove=(3, 13))
```

Comment:
Let's keep it as commented code not to confuse code readers which line is edited
```suggestion
# python-deadlib: Remove deprecation warning
# warnings._deprecated(__name__, remove=(3, 13))
```

Reply:
I am sorry to change words.

I originally thought users are expected to manually suppress deprecation warnings in user side, when importing it, because they already did it when using 3.12. This will also encourage users to look for another, better alternative of the dead libraries.
The purpose of the repository was providing CPython libraries via PyPI, rather than making users directly copying them from the old source code.

On the other hand, this is just a warning, which forever will exist, not actually changing the libraries' behavior.
Removing it will not break anything.

I'd like to listen from you about it.

Reply:
I'm not a fan of just changing the version number to something big.  As a result, users will get an warning of `aifc is deprecated and slated for removal in Python 3.100` which is just incorrect and unhelpful.

If you want a warning to show up to all users, I suggest using `warnings.warn(message, DeprecationWarning)` with a clear message about what exactly has happened, including the fact they're using standard-aifc from pip - not in python core.

Bear in mind, many people who see this mightn't be using the `aifc` module directly.  My project which hit a problem when upgrading to 3.13 actually uses `aifc` as a sub-sub-dependency.  I'd never heard of the module until I went investigating what had broken!

Reply:
Got it. `3.100` doesn't look good.
I am leaning to adding deprecation warning if you don't mind.
How did you handle it when you used 3.12? Did you suppress it when importing the library depending on aifc in its deep side?

Reply:
Thank you for confirming. Since it showed warnings for previous python versions too, I expect warnings from here doesn't make that much inconvenience.